### DISCUSSION TITLE
Welcome to Junicode-New Discussions!

### DISCUSSION THREAD

**Discussion by psb1558** [psb1558/Junicode-font#42] [General]
Since JuniusX/JuniusVF is more than usually complex, as fonts go, I've activated this discussion forum, where you can get answers to your questions about how to access its many characters, how to use its OpenType features, and more.

To acquaint yourself with the font, I'd suggest visiting the [specimen page](https://psb1558.github.io/Junicode-New/), and if you're thinking of using it in a web page, my little essay about [searchable and accessible text](https://psb1558.github.io/Junicode-New/Searchability.html).

This is also the appropriate place to ask questions like “when will the italic be done,” “when will there be a release,” “how do I subset this monster for my web page,” and “how do I make this work in MS Word?”

If you want to request a feature or report a bug, please [open an issue](https://github.com/psb1558/Junicode-New/issues).

**Comment by adunning** [comment]
Thank you, Peter; this is such a wonderfully useful project! If you're offering, I for one would be interested to know more about your plans – the italic will be wonderful, and I'm crossing my fingers that it might even be possible to have optical versions for different sizes as you did for Elstob. No pressure, of course; you're already improving the state of the art.

**Reply by psb1558** [reply]
Thanks, Andrew. Work on the italic outlines is now approximately 67% complete, and I hope to have it finished before the end of 2021. Sorry, there will be no optical size axis for JuniusX/JuniusVF—unless a younger and much more energetic designer decides to get involved. The project of designing new optical size masters for a font nearly four times the size of Elstob is simply too daunting.

**Comment by flogrz** [comment]
I just stumbled upon this thread, having found this brilliant and good-looking font. I've actually already found two smaller issues yesterday, which I reported since I think continued development of Junicode matters—it's one of the best free fonts out there (see f.ex. [this German typography rating](https://www.teuderun.de/schriftarten/top-10/), where Junicode is ranked #2 in Antiqua-Fonts for body-text serif fonts!).

I'm also very much interested in optical sizes and think that this font just deserves them. As I am fairly new to designing fonts myself (though I've been interested in typography for a longer time), I was curious whether anyone has reached to you to about joining a project to design the optical sizes for Junicode in the last six years—maybe at least for the latin alphabet. 

I hope that this project becomes real one day!

**Reply by psb1558** [reply]
Thanks for your kind words about the font. I have two fonts with optical sizes (Elstob and Irradiant), so I know what's involved: at least two new masters would have to be created, each with 3,133 non-component glyphs. That's at least 6,266 glyphs that would have to be newly drawn.

This could be done, but I am a retiree, and my time for this kind of work is limited. So I have resolved (there's a note to this effect somewhere in the repo) that I will undertake no major new projects related to Junicode.

It could be done, of course: Junicode is Open Source. What I would suggest is to put together a team of like-minded persons, fork the Junicode repo, and put together the required masters. If you're feeling generous after all that, you could submit a PR to this repo, and I would make sure it was well known who supplied the optical sizes.
